spec/samples/ad_verification_extension_and_node.js

```diff
@@ -0,0 +1,79 @@
+export const adVerificationFromExtensionsNodeAndFromNode = `
```

Comment:
This variable name is unclear you should rename it to something like this: `adVerificationsInDedicatedNodeAndInExtensionsNode` , you can also indent the tag for a better readibility and remove unnecessary tags.